Test/git-issues/git-issue-3187.dfy.expect

```diff
@@ -1,3 +1,5 @@
-git-issue-3187.dfy(6,31): Error: no comma is allowed between provides and reveals clauses in an export declaration
-git-issue-3187.dfy(6,31): Error: rbrace expected
-2 parse errors detected in git-issue-3187.dfy
+git-issue-3187.dfy(6,20): Error: no comma is allowed between provides and reveals and extends clauses in an export declaration
+git-issue-3187.dfy(10,12): Error: a const field is initialized using ':=', not '='
```

Comment:
I would maybe change this into "should be initialized". Otherwise it could be wrongly read as: is currently initialized using ":=", whereas it actually should be initialized using "=".

Reply:
Done - and in similar places

Reply:
Thanks, I can see the changes elsewhere. The one for `git-issue-3187.dfy.expect` above seems to be still missing though.

Reply:
Huh, I fixed the parser but forgot some of the tests. I wonder why I did not see a failure. Done